src/jit/emit.h

```diff
@@ -1202,13 +1202,17 @@ class   emitter
         ssize_t         idacAmdVal;
     };
 
-    struct          instrDescCGCA  : instrDesc      // call with ...
+    struct          instrDescCGCA : instrDesc      // call with ...
     {
         VARSET_TP       idcGCvars;                 // ... updated GC vars or
         ssize_t         idcDisp;                   // ... big addrmode disp
         regMaskTP       idcGcrefRegs;              // ... gcref registers
         regMaskTP       idcByrefRegs;              // ... byref registers
         unsigned        idcArgCnt;                 // ... lots of args or (<0 ==> caller pops args)
+
+#ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING_ONLY
+        GCtype          idSecondRetRegGCType;      // ... GC type for the second return register.
```

Comment:
naming convention: you should use idc... like the rest, e.g., idcSecondRetRegGCType.

However, I would suggest you name similar to the existing _idGCref, and call it idcSecondRetRegGCref. Also, like the existing _idGCref, you should define it as a 2-bit bitfield. IMO, you should also add setters/getters, like idGCref().